imageroot/crowdsec.service

```
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+#
+# This systemd unit starts a crowdsec instance using Podman.
+# Most parts of this file come from podman-generate-systemd.
+#
+
+[Unit]
+Description=crowdsec server
+After=network.target
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/var/lib/nethserver/%N/state/environment
+WorkingDirectory=/var/lib/nethserver/%N/state
+SyslogIdentifier=%N
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%N.pid %t/%N.cid
+ExecStartPre=/usr/local/bin/runagent -m %N expand-configuration
+ExecStartPre=/usr/local/bin/runagent -m %N expand-smarthost
+ExecStartPre=mkdir -vp crowdsec_config
```

Other directives use absolute paths...

```suggestion
ExecStartPre=/bin/mkdir -vp crowdsec_config
```